modules/simple-payments/simple-payments.php

```diff
@@ -40,6 +40,9 @@ private function register_scripts_and_styles() {
 
 	private function register_init_hook() {
 		add_action( 'init', array( $this, 'init_hook_action' ) );
+		if ( function_exists( 'register_meta' ) ) {
```

Comment:
What's the purpose of checking for this function? When would you expect it not to be present?

Reply:
Just played safe without checking how long it has been available. Seems like since WordPress 3.3.0 so it's safe to remove the check.

https://developer.wordpress.org/reference/functions/register_meta/